README.md

```diff
@@ -15,19 +15,18 @@ The edited content will be posted to Slack:
 
 ## Installing
 
-1. To get automated posting to Slack, first install [Slackistrano](https://github.com/phallstrom/slackistrano).
-2. Add capistrano-fiesta to your application's Gemfile:
+1. Add capistrano-fiesta to your application's Gemfile:
 
   ```ruby
   gem 'capistrano-fiesta'
   ```
-3. Require in the capfile or appropriate stage and configure the Slack channel:
+2. Require in the capfile or appropriate stage and configure the Slack channel:
 
   ```ruby
   require 'capistrano-fiesta'
   set :fiesta_slack_channel, '#release'
```

Comment:
I think we have some extra required fields now, that would normally have been configured as part of installing `Slackistrano`:

https://github.com/balvig/capistrano-fiesta/blob/18389ee1ba9366b98630182f3b00e3e11e1f1771/lib/capistrano/tasks/fiesta.rake#L65-L67

Could we add those to the installation docs?

Reply:
0f003a6 🔨 

I have dropped support for `slack_team` and `slack_token` variables as the incoming webhook URL format using these variables is not documented in the Slack's documentation and we should always be able to use the `webhook` variable.